packages/@aws-cdk/aws-elasticloadbalancingv2/test/alb/test.listener.ts

```diff
@@ -673,6 +673,72 @@ export = {
     test.done();
   },
 
+  'Can add simple redirect responses'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const vpc = new ec2.Vpc(stack, 'VPC');
+    const lb = new elbv2.ApplicationLoadBalancer(stack, 'LoadBalancer', {
+      vpc,
+    });
+
+    // WHEN
+    lb.addRedirect({});
```

Comment:
_Nit_: Slightly more idiomatic usage:
```suggestion
    lb.addRedirect();
```